src/routes.js

```diff
@@ -9,6 +9,9 @@ export const routes = [
   {
     path: "",
     component: Home,
+    meta: {
+      keepAlive: true
```

Comment:
Not too sure about this myself. I'm interested in hearing @MartonDev's comments about this. Super creative solution @HermitSun!

Reply:
@martonlederer 😉